9.4/root/usr/libexec/fix-permissions

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+# Fix permissions on the given directory to allow group read/write of
+# regular files and execute of directories.
+chown -R postgres "$1"
+chgrp -R 0 "$1"
+chmod -R g+rw "$1"
+find "$1" -type d -exec chmod g+x {} +
```

Comment:
it's not in our sti-base script either:
https://github.com/openshift/sti-base/blob/master/bin/fix-permissions

updating this script here and elsewhere can be a follow up.